docs/using.rst

```diff
@@ -473,7 +473,7 @@ does include the nginx plugin package:
    emerge -av app-portage/layman
    layman -S
    layman -a mrueg
-   emerge -av app-crypt/letsencrypt-nginx
+   emerge -av app-crypt/certbot-nginx
```

Comment:
Like 3rd party plugins, these instructions should not be changed until we know when/if/how they will be changed.